analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/RoslynSymbolicExecutionTest.IsNull.cs

```diff
@@ -215,15 +215,20 @@ public void IsNull_CoalesceAssignment_UnknownToNull()
         [TestMethod]
         public void IsNull_CoalesceAssignment_UnknownToNotNull()
         {
-            const string code = @"
-object notNullValue = new object();
-arg ??= notNullValue;
-Tag(""Arg"", arg);";
-            var validator = SETestContext.CreateCS(code, ", object arg").Validator;
+            const string code = """
+                object notNullValue = new object();
+                arg ??= notNullValue;
+                Tag("End");
+                """;
+            var validator = SETestContext.CreateCS(code, ", object arg", new PreserveTestCheck("arg", "notNullValue")).Validator;
```

Comment:
`new PreserveTestCheck("arg", "notNullValue")` is the actual fix. I also changed the assertion, to make sure that both variables have the right constraints set.